.eslintrc.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * © 2017 Liferay, Inc. <https://liferay.com>
```

Comment:
Should this be `2019`?

Reply:
According to [this guidance](https://github.com/liferay/clay/pull/1451#issuecomment-456384580):

> just change the year to whatever year that file was in (if that’s too much work, `2014` as the year of the first commit should do). And then do not bother with updating the year in the future.

As the first commit was in 2017, that's the date I used.

Reply:
LGTM